apps/frontend/src/app/2_molecules/EmailContentRenderer/EmailContentRenderer.tsx

```diff
@@ -0,0 +1,33 @@
+import React from 'react';
+
+import { Helmet } from 'react-helmet-async';
+
+import { Heading, Paragraph } from '@sovryn/ui';
+
+type EmailContentRendererProps = {
+  metaTitle: string;
+  title: string;
+  subtitle: React.ReactNode;
+};
+
+export const EmailContentRenderer: React.FC<EmailContentRendererProps> = ({
```

Comment:
The name of the component is really misleading - you don't render any email content with it. I think something like `EmailVerificationState` is more suitable.